toolchain/testing/file_test.cpp

```diff
@@ -193,6 +193,8 @@ auto ToolchainFileTest::GetDefaultArgs() const
     args.push_back("--dump-parse-tree");
   } else if (component_ == "check") {
     args.insert(args.end(), {"--dump-sem-ir", "--dump-sem-ir-ranges=only"});
+  } else if (component_ == "codegen") {
+    // codegen tests specify flags as needed.
```

Comment:
Seems like this should be after "lower"?

Reply:
Done